src/index.js

```diff
@@ -142,10 +151,37 @@ function init( {
 					} else {
 						node.addEventListener( 'mouseenter', showPopup )
 					}
+
+					foundDetectLinks.push( {
+						text: node.textContent,
+						title: matches.title,
+						lang: matches.lang
+					} )
 				}
 			}
 		)
 	}
+
+	if ( debug ) {
+		/* eslint-disable no-console */
+		console.group( 'Wikipedia Preview [debugger]' )
```

Comment:
`debugger` exists as a specific instruction in javascript, maybe use `[debug mode]` instead to avoid confusion.